authz.go

```diff
@@ -240,6 +244,10 @@ func (p Permission) matchesV1(perm Permission) bool {
 		return false
 	}
 
+	if p.Resource.Type == InstanceResourceType {
```

Comment:
Is `MATCHER_BEHAVIOUR` documented anywhere? Can we remove it?

Reply:
If it's an undocumented environment variable for OSS, I'd lean toward assuming nobody has it set, and cleaning up the code.

We can mention the behaviour change in the release notes in case someone is depending on this undocumented behaviour, but I expect this is a relic of the Cloud/OSS split.

Reply:
@jeffreyssmith2nd Make sure we have a docs issue for this.